### ISSUE TITLE
Parallel computing tasks

### ISSUE THREAD

**Issue by mrdcherrie** [GoogleCloudPlatform/datalab#775]
Hey, is it possible for you to include the clusters tab within datalab? Along with easy collaboration, parallel processing using ipyparallel, would truly unlock the benefits of running data experiments within the notebook on the compute engine. Right now I have three of the four cores doing nothing while running random forests regression.

For now, I am building my own container, should I modify the dependencies listed in the Docker.in file

like so:
pip install -U ipyparallel && \

Will there be a problem? 

Also, what is the best way to deal with HDF5?

Thanks,

Daniel

**Comment by gramster** [comment]
If you want to use that in a notebook, you can just have a cell in your notebook with:

    %%bash
    pip install --upgrade ipyparallel

**Comment by mrdcherrie** [comment]
I know that is possible, but its a workaround and not permanent. As for HDF5? Its not possible to simply pip install that module as it relies on other installations.

**Comment by gramster** [comment]
Yes, it isn't permanent, but unless we have many users clamoring for a library it is unlikely to be added to the default container. As you note, you can build your own custom container.

**Comment by gramster** [comment]
I hadn't use this before. Doing the pip install is not enough. We have a customized UX that will not show a clusters tab.

**Comment by VelizarVESSELINOV** [comment]
+1

**Comment by joshreuben456** [comment]
👍
